tests/services/test_file_thawer.py

```
@@ -180,3 +187,65 @@ def test_thaw_file_not_found_in_cold(self, db_session, tmp_path):
+            path_id=inv.path_id,
+            original_path=str(hot_file),
+            cold_storage_path=cold_ref,
+            cold_storage_location_id=storage_location.id,
+            file_size=23,
+            operation_type=OperationType.MOVE,
+        )
+        db_session.add(record)
+        db_session.commit()
+
+        class DummyBackend:
+            def backend_name(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `backend_name`

Add return type annotation: `str`

(ANN202)